tests/ts/liblastlog2/dlopen

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+TS_TOPDIR="${0%/*}/../.."
+TS_DESC="dlopen"
+
+. "$TS_TOPDIR"/functions.sh
+ts_init "$*"
+
+if [ -z "${top_builddir}" ]; then
+  # This should never happen
+  exit 1
```

Comment:
Otherwise shellcheck would blame that this variable has not been initialized.
The other scripts in tests/ts have the same problem, but shellcheck will be called only if the source code has been
changed. So, is my way OK for you ?

Reply:
Shellcheck is wrong here. I would prefer to just drop the check.

Reply:
How can I do this ? It will be called in
https://github.com/util-linux/util-linux/actions/runs/6587935047/job/17899179605?pr=2508

Reply:
There will be a warning but it won't block the merge.